### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-it/content_conversationlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#225]
```
Added to italian translation Conversationlist
conversationlist_jolnor
conversationlist_alynndir
conversationlist_shops
conversationlist_ogam
conversationlist_foamingflask
conversationlist_ambelie
conversationlist_foamingflask_guards
conversationlist_foamingflask_outsideguard
conversationlist_wrye
```

Original issue reported on code.google.com by `k6b...@gmail.com` on 28 Jul 2011 at 11:07

Attachments:
- [content_conversationlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-225/comment-0/content_conversationlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, committed in r144.
```

Original comment by `oskar.wi...@gmail.com` on 30 Jul 2011 at 12:13
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.10**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Issue 222 has been merged into this issue.
```

Original comment by `oskar.wi...@gmail.com` on 30 Jul 2011 at 12:14

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
